src/main/java/rockset/models/KafkaMessage.java

```diff
@@ -15,6 +15,7 @@ public class KafkaMessage {
   @SerializedName("partition") public int partition;
   @SerializedName("offset") public long offset;
   @SerializedName("key") public Object key;
+  @SerializedName("timestamp") public long timestamp;
```

Comment:
we need changes in https://github.com/rockset/rs/blob/1a4e2dd8b96a3df23a41a4377b98fb34e9f63071/java/rockset-main/src/main/java/io/rockset/main/api/models/documents/KafkaMessage.java#L7-L8 too.

And I don't think we can change the kafka-connect code until receiver understands this new field

Reply:
Shouldn't we have connect populate this field first as it is upstream? You mean there will be a parse error?

Reply:
receiver will reject unknown fields from what I know. Should be easy to verify